pkg/cmd/auth/shared/oauth_scopes_test.go

```diff
@@ -11,6 +11,54 @@ import (
 )
 
 func Test_HasMinimumScopes(t *testing.T) {
+	tests := []struct {
```

Comment:
broke up the existing tests from `Test_HasMinimumScopes` to a single pass and fail with `httpmock`, and copied the rest below as unit tests to `Test_HeaderHasMinimumScopes`